src/contrast/pages/UploadNewPaper/ExperimentsSection/Findings.jsx

```diff
@@ -148,7 +147,15 @@ export default function Findings({
                           setFieldValues(newArray);
                         }}
                         options={alphabetizeByLabels(
-                          fieldOptions.techniquesOptions
+                          // triming the list of option so no duplicated options
+                          fieldOptions.techniquesOptions.filter(
+                            (obj, index, self) =>
```

Comment:
What is this supposed to do? It's not clear

Reply:
its written in comment on the row above